plugins/woocommerce-admin/client/customize-store/assembler-hub/style.scss

```diff
@@ -427,3 +473,15 @@
 		margin-left: 12px;
 	}
 }
+
+.woocommerce-customize-store_global-styles-variations_item {
+	border: 1px solid transparent;
+
+	&:hover,
+	&.is-active {
+		border-radius: 2px;
+		border: 1px solid #dcdcde;
+		background: #fff;
+		box-shadow: 0 0 0 4px  var(--wp-admin-theme-color), 0 0 0 2.5px #fff;
```

Comment:
I think the selected border is a bit different compared to design


<img width="127" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/56898cf0-7a00-4594-a27f-fa2288acf4ab">


<img width="120" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/aa223e6a-99ec-49b9-9bdd-b0cbaea6c0d9">

Reply:
Good catch! 

It looks like the styles provided by figma need to be corrected, fixed the border style in 16deb7625a658260cb33176d3b35e5caedbb8fde.